src/ml/neural_net/tf_compute_context.cpp

```diff
@@ -28,7 +33,9 @@ static auto* tf_registration = new compute_context::registration(
 }  // namespace
 
 tf_compute_context::tf_compute_context()
-{}
+{
```

Comment:
Nit: why change this? If we change this, may as well use the modern syntax:
```
tf_compute_context::tf_compute_context() = default;
```

Reply:
Will change it to the modern syntax.